### PR TITLE
add a feature flag for the cluster class ui support

### PR DESCRIPTION
**Pull request by raymondz1**
### Which issue(s) this PR fixes

Fixes #

### Release note

```release-note

```

### PR Checklist

- [ ] Squash the commits into one or a small number of logical commits
      
- [ ] Use good commit [messages](https://github.com/vmware-tanzu/tanzu-framework/blob/main/CONTRIBUTING.md)
- [ ] Ensure PR contains terms all contributors can understand and links all contributors can access

### CODE REVIEW

File: pkg/v1/tkg/web/node-server/src/routes/api/mockService.js [hunk 1]
```diff
@@ -69,6 +69,7 @@ router.get(`${ENDPOINT}/features`, (req, res) => {
             "import": "true",
             "vsphereIPv6": "true",
             "standalone-cluster-mode": "false",
+            "clusterClassSupport": "true"
```
Comment (swalner-vmware) [inline pkg/v1/tkg/web/node-server/src/routes/api/mockService.js:72]: I think most of the flags use the hyphenated case, so cluster-class-support might be more consistent. However, I have no strong opinion

**Review by swalner-vmware — APPROVED** [review]

File: pkg/v1/tkg/web/node-server/src/routes/api/mockService.js (hunk 1 above)
Comment (miclettej) [reply pkg/v1/tkg/web/node-server/src/routes/api/mockService.js:72]: +1

**Review by miclettej — APPROVED** [review]
Just one minor change then good to go

File: pkg/v1/tkg/web/node-server/src/routes/api/mockService.js (hunk 1 above)
Comment (raymondz1) [reply pkg/v1/tkg/web/node-server/src/routes/api/mockService.js]: renamed

**Pull request merged**